File: led.c
#include <wiringPi.h>
#include <stdio.h>
#include <stdlib.h>

int main(void) {
  int ret;
  int leds[] = { 7, 21, 22, 10, 11, 13, 12, 1 };
  int leds_num = 8;
  int led;
  int i;

  ret = wiringPiSetup();
  if (ret < 0) {
    printf("wiringPiSetup(): error initializing wiringPi\n");
    exit(-1);
  }

  printf("------\n");
  for (i = 0; i < leds_num; i++) {
    // LED index to BMC pin
    led = leds[i];

    printf("Settting GPIO %d as OUTPUT\n", led);
    pinMode(led, OUTPUT);
  }
  printf("------\n");

  while(1) {
    for (i = 0; i < leds_num; i++) {
      led = leds[i];

      digitalWrite(led, LOW);
      printf("GPIO %d is ON\n", led);
      delay(100);
      digitalWrite(led, HIGH);
      printf("GPIO %d is OFF\n", led);
      delay(100);
    }
  }
}

File: led.py
#!/bin/python
import RPi.GPIO as GPIO
import time

interval = 0.1

# See passthrough.lpf for data_in[7:0] locates onto pins
#   LED 0, 1, 2, ...
leds = [4, 5, 6, 8, 7, 9, 10, 18]

print "------"
for ld in leds:
    print "Setting GPIO %d as output..." % ld
    GPIO.setmode(GPIO.BCM)
    GPIO.setwarnings(False)
    GPIO.setup(ld, GPIO.OUT)

print "------"

while (1):
    for ld in leds:
        print("GPIO %d on" % ld)
        GPIO.output(ld, GPIO.LOW)
        time.sleep(interval)

        print "GPIO %d off" % ld
        GPIO.output(ld, GPIO.HIGH)
        time.sleep(interval)
